samples/Mvc.Client/Views/Authentication/SignIn.cshtml

```diff
@@ -1,16 +1,17 @@
-﻿@using Microsoft.AspNetCore.Http.Authentication
-@model IEnumerable<AuthenticationDescription>
+﻿@using Microsoft.AspNetCore.Authentication
+@model AuthenticationScheme[]
```

Comment:
`IEnumerable<AuthenticationScheme>` seems nicer

Reply:
FWIW, I also used `AuthenticationScheme[]` for the ASOS client sample: https://github.com/aspnet-contrib/AspNet.Security.OpenIdConnect.Server/blob/dev/samples/Mvc/Mvc.Server/Views/Shared/SignIn.cshtml#L2.

For consistency, I'd say let's use an array here too (plus it avoids an allocation as the `foreach` is supposed to be optimized to avoid using `GetEnumerator()` when the iterated collection is an array known at compile time :sweat_smile:).

Reply:
Where's the puke emoji :frowning:

Reply:
@davidfowl instead, should we inject `UserManager` and do some async voodoo magic directly in the view like in the Identity templates? :trollface: 

https://github.com/aspnet/Templates/blob/dev/template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Views/Account/Login.cshtml#L6